tests/preprocessing/input_output/aminoseq.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from rdkit.Chem import Mol, MolFromFASTA, MolToSmiles
+
+from skfp.preprocessing import MolFromAminoseqTransformer
+
+
+@pytest.fixture
+def fasta_list():
+    return [
+        ">peptide_pm_1\nKWLRRVWRWWR\n",
+        ">peptide_pm_2\nFLPAIGRVLSGIL\n",
+        ">peptide_pm_3\nCGESCVWIPCISAVVGCSCKSKVCYKNGTLP\n",
+        ">peptide_pm_4\nILGKLLSTAWGLLSKL\n",
+        ">peptide_pm_5\nWKLFKKIPKFLHLAKKF\n",
+        ">peptide_pm_6\nRAGLQFPVGRLLRRLLRRLLR\n",
+        ">peptide_pm_7\nGLWSKIKTAGKSVAKAAAKAAVKAVTNAV\n",
+        ">peptide_pm_8\nCGESCVYIPCLTSAIGCSCKSKVCYRNGIP\n",
+    ]
+
+
+@pytest.fixture
+def sequence_list(fasta_list):
+    return [fst.split("\n")[1] for fst in fasta_list]
+
+
+@pytest.fixture
+def peptide_list(fasta_list):
+    return [MolFromFASTA(fst) for fst in fasta_list]
+
+
+@pytest.fixture
+def peptide_smiles(peptide_list):
```

Comment:
how about smiles_list

Reply:
To be precise, we would need `peptides_smiles_list`, since `smiles_list` exists